python/ycm/client/completion_request.py

```diff
@@ -190,7 +190,27 @@ def _GetCompletionInfoField( completion_data ):
 
   # This field may contain null characters e.g. \x00 in Python docstrings. Vim
   # cannot evaluate such characters so they are removed.
-  return info.replace( '\x00', '' )
+  info = info.replace( '\x00', '' )
+
+
+  vf_strdisplaywidth = vimsupport.vim.Function('strdisplaywidth')
+  vf_winwidth = vimsupport.vim.Function('winwidth')
```

Comment:
Calling this function might have some overhead, We should probably call it for each vim function just once in an init of a python class.